shared-libs/message-utils/src/index.js

```diff
@@ -345,26 +345,51 @@ exports.generate = function(config, translate, doc, content, recipient, extraCon
 exports.template = function(config, translate, doc, content, extraContext) {
   extraContext = extraContext || {};
   const locale = getLocale(config, doc);
-  let template;
-  if (content.translationKey) {
-    template = translate(content.translationKey, locale);
-  } else if (_.isArray(content.message)) {
-    const message = _.find(content.message, { locale: locale }) ||
-                  content.message[0];
-    if (message) {
-      template = message.content && message.content.trim();
-    }
-  } else {
-    // depecated - already generated message
-    template = content.message;
-  }
+  const template = exports.getMessage(content, translate, locale);
+
   if (!template) {
     return '';
   }
+
   const context = extendedTemplateContext(doc, extraContext);
   return render(config, template, context, locale);
 };
 
+/*
+ * Take message configuration and return message content. The configuration
+ * should have either a `messages` property with an array of messages, or
+ * a `translation_key` property with a string.
+ * Use locale if found otherwise defaults to 'en'.
+ */
+exports.getMessage = function(configuration, translate, locale, logger) {
+  if (!configuration) {
+    return '';
+  }
+  // use the translation key if provided
+  if (configuration.translation_key) {
+    return translate(configuration.translation_key, locale);
+  }
+
+  // use the translation key if provided
+  if (configuration.translationKey) {
+    return translate(configuration.translationKey, locale);
+  }
```

Comment:
Combine these branches, eg:

```suggestion
  // use the translation key if provided
  const translationKey = configuration.translation_key || configuration.translationKey;
  if (translationKey) {
    return translate(translationKey, locale);
  }
```